test_requirements.txt

```diff
@@ -6,6 +6,7 @@ hypothesis==6.24.1
 pytest==6.2.5
 pytz==2021.3
 pytest-cov==3.0.0
+pytest-xdist
```

Comment:
Do you want to keep this as a project-wide requirement?

Reply:
Probably, since otherwise the -j option doesn't work. But I don't feel particularly strongly about it either way.